### PR TITLE
updated dialogs (no contact info for staff, cleanup)

### PR DESCRIPTION
**Pull request by severinbeauvais**
*Issue #:* /bcgov/entity#1733

*Description of changes:*

- hid contact info for staff in dialogs
- updated some dialog text per wireframes
- changed Okay buttons to OK (per Scott)
- aligned dialog buttons consistently
- made all dialogs persistent (must click button to close)
- made Save Error dialog narrower (per Scott)
- added IDs to dialogs
- updated unit tests
- fixed missing enum rename
- changed Dashboard responsive breakpoint to medium (per Scott) (no change to main views/pages)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: coops-ui/src/components/AnnualReport/PaymentErrorDialog.vue [hunk 1]
```diff
@@ -1,38 +1,54 @@
 <template>
-  <v-dialog v-model="dialog" width="45rem">
+  <v-dialog id="payment-error-dialog" v-model="dialog" width="45rem" persistent>
```
Comment (cameron-freshworks) [inline coops-ui/src/components/AnnualReport/PaymentErrorDialog.vue]: I like the addition of `persistent` tho to be honest, i would have assumed it to be default! Good job.

File: coops-ui/src/components/AnnualReport/SaveErrorDialog.vue [hunk 2]
```diff
@@ -44,20 +44,28 @@
         </v-card-actions>
         <v-card-actions v-else>
           <v-spacer></v-spacer>
-          <v-btn id="okay-btn" color="primary" text @click="okay()">Okay</v-btn>
+          <v-btn id="okay-btn" color="primary" text @click="okay()">OK</v-btn>
```
Comment (cameron-freshworks) [inline coops-ui/src/components/AnnualReport/SaveErrorDialog.vue]: Love the verbiage clarification ;)

Comment (severinbeauvais) [reply coops-ui/src/components/AnnualReport/SaveErrorDialog.vue]: This is the way Scott wants it :)

**Comment by severinbeauvais** [comment]
> Looks good to me 👍
> 
> Just to clarify, what is the use case of the id's in the dialogs, is it mainly for testing?
> 
> Also, would this be a good time to reshuffle the location of the dialogs into a `Dialogs` folder, inside the common directory, use an index for cleaning imports app wide? It would require some import refactoring in quite a few components so it may fall outside the scope of this ticket, but just curious on your thoughts.
> 
> Approved!

I am working on a sub-ticket of /bcgov/entity#1721 which is specifically adding IDs to facilitate testing. My approach is:
- Things that should have IDs anyways will get `id=...` (and we can use those in our unit tests).
- Things that mainly require IDs for QA will get `data-test-id=...`. All our code should work without these (say, by stripping them all out with a preprocessor).

Actually, adding IDs to the v-dialogs won't work (because the dialogs are attached outside the components). But I figured out how to do this... and will refactor in my next PR!

My **next** PR will move the dialog components to a 'dialogs' folder! Great minds think alike :)

**Pull request merged**